packages/bpk-tokens/src/android/base/typography.json

```diff
@@ -20,10 +20,22 @@
     },
     "TEXT_COLOR_PRIMARY_LIGHT": {
       "value": "{!GRAY_900}",
+      "deprecated": true,
```

Comment:
Given it was so short lived do we assume a consumer has started to use these values? As I would rather we delete than just deprecate

But if not we will remove it in the future anyway

Reply:
Let's remove in next breaking

Reply:
Sounds good to me 👍